1-select-tasks/4.txt

```diff
@@ -2,4 +2,14 @@ Create a query to return:
 
 | Customer Id | Total number of Orders | % of all orders |
  
-order data by % - higher percent at the top.
\ No newline at end of file
+order data by % - higher percent at the top.
+
+--task4
+DECLARE @var DECIMAL;
+SET @var = (SELECT COUNT([o].[OrderID])
+FROM [dbo].[Orders] [o])
+SELECT [o].[CustomerID],  COUNT([o].[CustomerID]) AS [Total number of Orders], COUNT([c].[CustomerID])/@var*100 AS [ % of all orders]
```

Comment:
Not what i meant. See what literals([aka](http://dictionary.cambridge.org/dictionary/english/aka). constants) are. ([link](https://en.wikipedia.org/wiki/Literal_(computer_programming)))
![image](https://user-images.githubusercontent.com/7272208/30251959-c3f47940-9672-11e7-9622-c6872bfb191b.png). What literals do you define in your query?

When you use variable please define meaningful name for it.